assets/sparksql/search.py

```
@@ -0,0 +1,19 @@
+# executed within container
+import sys
+import os
+
+from pyspark.sql import SparkSession
+
+spark = SparkSession \
+    .builder \
+    .appName("clp-bench") \
+    .master("spark://localhost:7077") \
+    .config("spark.sql.caseSensitive", True) \
```

Ruff (0.12.2)

Boolean positional value in function call

(FBT003)